ruby_on_rails/rails_basics/views.md

```diff
@@ -230,7 +230,7 @@ This section contains helpful links to other content. It isn't required, so cons
 * [Stack Overflow Post on Views](https://stackoverflow.com/questions/14429910/an-alternate-explanation-to-rails-layouts-rendering-partials-templates-and-v)
 * [Video on the Relationship Between Views and Controllers](https://www.youtube.com/watch?v=mRJSovhdzWc&ab_channel=GoRails)
 * [Video on ERB Tags](https://www.youtube.com/watch?v=na28woOGPUw&ab_channel=NoobandTube) - (this video will require you to turn your volume up)
-* [Introduction to Ruby on Rails from freeCodeCamp](https://youtu.be/fmyvWz5TUWg) - (first 60 minutes is recommended)
+* [Introduction to Ruby on Rails from freeCodeCamp](https://youtu.be/fmyvWz5TUWg) - If you'd like a deeper dive into routing, MVC, CRUD and partials watch the first 60 minutes of this video which provides a good warp up for what we've been learning so far, you'll get to create a simple Rails app from scratch.
```

Comment:
@ChargrilledChook it's right here.

Reply:
Thanks @egemen-github , looks good. Two minor things - minor typo (warp -> wrap) and I think we should break the sentence up a little bit. What do you think?

```suggestion
* [Introduction to Ruby on Rails from freeCodeCamp](https://youtu.be/fmyvWz5TUWg) - If you'd like a deeper dive into routing, MVC, CRUD and partials watch the first 60 minutes of this video which provides a good wrap up for what we've been learning so far. You'll also get to create a simple Rails app from scratch if you'd like some extra practice.
```

Reply:
I agree. I think it is better with the suggested changes.